nuxeo-runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/computation/TestStreamProcessor.java

```diff
@@ -106,7 +106,7 @@ public void testSimpleTopo(int nbRecords, int concurrency) throws Exception {
                 long lowWatermark = processor.getLowWatermark();
                 log.info("low: " + lowWatermark + " dist: " + (targetWatermark - lowWatermark));
             }
-            double elapsed = (double) (System.currentTimeMillis() - start) / 1000.0;
+            double elapsed = (System.currentTimeMillis() - start) / 1000.0;
```

Comment:
Not sure why you backport cleanups, this doesn't help reading the code.

Reply:
> Not sure why you backport cleanups, this doesn't help reading the code.

it's because I did not cherry-picked a given commit. I use the code from master to update the classes from 10.10, so yes it includes more than just the Probe feature. However is it bad to keep the cast cleanup? I guess it will ease potential backports